### PR TITLE
feat(regression github): send failed tests names to chat 

### PR DESCRIPTION
**Pull request by adiholden**

### CODE REVIEW

File: tests/dragonfly/requirements.txt [hunk 1]
```diff
@@ -14,3 +14,5 @@ pytest-asyncio==0.20.1
 pytest-repeat==0.9.1
 pymemcache==4.0.0
 numpy==1.24.3
+pytest-json-report
```
Comment (dranikpg) [inline tests/dragonfly/requirements.txt:17]: Don't you have to specify at least some lower bound version requirement?

**Review by dranikpg — DISMISSED** [review]

File: .github/workflows/regression-tests.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Configure & Build
         run: |
-          apt update && apt install -y pip
+          apt update && apt install -y pip  && apt install -y jq
```
Comment (royjacobson) [inline .github/workflows/regression-tests.yml:34]: small nit -
```suggestion
          apt update && apt install -y pip jq
```

**Review by royjacobson — APPROVED** [review]

File: .github/workflows/regression-tests.yml (hunk 2 above)
Comment (adiholden) [reply .github/workflows/regression-tests.yml:34]: changing it to apt update && apt install -y pip jq

File: tests/dragonfly/requirements.txt (hunk 1 above)
Comment (adiholden) [reply tests/dragonfly/requirements.txt:17]: I dont have to. I we dont specify a version the latest one will be taken.
Adding a version is for compatibility issues for later version.
I will add a version

**Review by dranikpg — APPROVED** [review]

**Pull request merged**